doc/news/changes/minor/20200326NiklasFehn

```diff
@@ -0,0 +1,7 @@
+New: There is a new function ParameterHandler::parse_input() that
```

Comment:
Do you mean
```suggestion
New: There is a new function ParameterHandler::parse_input_any_type() that
```

Reply:
I updated this. I also renamed ``_all_entries_`` into ``_mandatory_entries_`` since "all" can be misunderstood.